# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cdrdao/make-more.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./makefile.org	Sat Dec 28 23:21:18 2002
+++ ./makefile	Sat Dec 28 23:21:31 2002
@@ -50,10 +50,10 @@
 	@echo antlr executable now in $(BINDIR)
 	(cd ./dlg; $(MAKE) CC="$(CC)" COPT="$(COPT)")
 	@echo dlg executable now in $(BINDIR)
-#	(cd ./sorcerer; $(MAKE) CC="$(CC)" COPT="$(COPT)")
-#	@echo sorcerer executable now in $(BINDIR)
-#	(cd ./support/genmk; $(MAKE) CC="$(CC)" COPT="$(COPT)"; mv genmk ../../$(BINDIR))
-#	@echo genmk executable now in $(BINDIR)
+	(cd ./sorcerer; $(MAKE) CC="$(CC)" COPT="$(COPT)")
+	@echo sorcerer executable now in $(BINDIR)
+	(cd ./support/genmk; $(MAKE) CC="$(CC)" COPT="$(COPT)"; mv genmk ../../$(BINDIR))
+	@echo genmk executable now in $(BINDIR)
 	@echo
 	@echo "       PCCTS 1.33MR32 installation complete"  # MRXXX
 
